src/databricks/labs/ucx/source_code/jobs.py

```diff
@@ -139,11 +139,12 @@ def _register_spark_python_task(self, graph: DependencyGraph):  # pylint: disabl
         path = WorkspacePath(self._ws, notebook_path)
         return graph.register_notebook(path)
 
-    def _register_python_wheel_task(self, graph: DependencyGraph):  # pylint: disable=unused-argument
+    def _register_python_wheel_task(
+        self, graph: DependencyGraph
+    ) -> Iterable[DependencyProblem]:  # pylint: disable=unused-argument
         if not self._task.python_wheel_task:
             return
-        # TODO: https://github.com/databrickslabs/ucx/issues/1640
-        yield DependencyProblem('not-yet-implemented', 'Python wheel task is not yet implemented')
+        yield from graph.register_import(self._task.python_wheel_task.package_name.replace("_", ".").lower())
```

Comment:
A proper approach would be to locate the entrypoint using [`importlib.metadata.entrypoints`](https://docs.python.org/3/library/importlib.metadata.html#entry-points), then use the entrypoint module as starting point for linting.

However, this requires `importlib` to be able to locate the wheel, which it can't because we "installed" it in our own path location. An approach could be to extend the `sys.meta_path` with a finder for our path